packages/back-end/src/routers/sdk-connection/sdk-connection.controller.ts

```diff
@@ -191,6 +194,39 @@ export const checkSDKConnectionProxyStatus = async (
   });
 };
 
+export const getSDKConnectionsWebhooks = async (
+  req: AuthRequest,
+  res: Response<{
+    status: 200;
+    connections: Record<string, WebhookSummary[]>;
+  }>
+) => {
+  const context = getContextFromReq(req);
+  const connections = await findSDKConnectionsByOrganization(context);
+  const connectionIds = connections.map((conn) => conn.id);
+
+  const allWebhooks = await findAllSdkWebhooksByConnectionIds(
+    context,
+    connectionIds
+  );
+
+  // Group webhooks by connection ID
+  const webhooksByConnection: Record<string, WebhookSummary[]> = {};
+  connections.forEach((connection) => {
+    const webhooks = allWebhooks
+      .filter((w) => w.sdks.includes(connection.id))
+      .map((w) =>
+        pick(w, ["id", "name", "endpoint", "lastSuccess", "error", "created"])
+      );
+    webhooksByConnection[connection.id] = webhooks;
+  });
```

Comment:
Nit: This is `O(n^2)`, if you loop over `allWebhooks` instead it will be `O(n)`.  Not a big deal in this case since the numbers are small.